projects/plugins/jetpack/modules/sso.php

```diff
@@ -679,6 +679,16 @@ public function jetpack_user_table_styles() {
 			.jetpack-sso-invitation.sso-disconnected-user:active {
 				color: #0096dd;
 			}
+			.tooltip {
+				background: #aaaaaa;
```

Comment:
This is a sass variable that comes from `@automattic/jetpack-base-styles/root-variables` since this is a CSS style (not SCSS file), that would require some workaround to have it available. For just a plain white color, I think It is not worth.

Reply:
You're right 👍